src/graphics/webgl/webgl-graphics-device.js

```diff
@@ -1270,7 +1270,8 @@ class WebglGraphicsDevice extends GraphicsDevice {
                     gl.renderbufferStorage(gl.RENDERBUFFER, gl.DEPTH_STENCIL, target.width, target.height);
                     gl.framebufferRenderbuffer(gl.FRAMEBUFFER, gl.DEPTH_STENCIL_ATTACHMENT, gl.RENDERBUFFER, target._glDepthBuffer);
                 } else {
-                    gl.renderbufferStorage(gl.RENDERBUFFER, gl.DEPTH_COMPONENT16, target.width, target.height);
+                    const depthFormat = this.webgl2 ? gl.DEPTH_COMPONENT32F : gl.DEPTH_COMPONENT16;
```

Comment:
wouldn't gl.DEPTH_COMPONENT24 for webgl2 match the usual (backbuffer) case better?

Reply:
slightly concerned about unnecessary perf cost using fp32

Reply:
And also, even 24 bit buffer would internally be stored as 32bit, so there is no size / bandwidth win either.

Reply:
I would assume float slower than int, especially on mobile. Yes you're right bandwidth is the same between 24 and 32bit, but float could still be slower than int on mobile.